network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/publication_hero.html

```diff
@@ -4,7 +4,7 @@
   {% image page.hero_image fill-1400x675 as bg_img %}
   <div
     class="d-flex align-items-center publication-hero-container{% if size %} small{% endif %}"
-    style='background: linear-gradient(rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)), url("{{ bg_img.url }}")'>
+    style='background-image: linear-gradient(rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)), url("{{ bg_img.url }}")'>
```

Comment:
along with changes in https://github.com/mozilla/foundation.mozilla.org/pull/6424/files#diff-e8849b57dac5efaf9860f99be5bb936b1efc614d17faa1be032d781c7dcc6d80

Reply:
Test pages

- publication page: https://foundation-s-issue-6355-xnjp28.herokuapp.com/en/publication-page-with-child-article-pages/
- article page: https://foundation-s-issue-6355-xnjp28.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/